packages/core-components/src/components/MarkdownContent/MarkdownContent.tsx

```diff
@@ -88,14 +90,17 @@ const components: Options['components'] = {
  * MarkdownContent
  * --
  * Renders markdown with the default dialect [gfm - GitHub flavored Markdown](https://github.github.com/gfm/) to backstage theme styled HTML.
+ * Pass html to activate [rehypeRaw](https://github.com/remarkjs/react-markdown#appendix-a-html-in-markdown) styling that will render any inlined HTML in gfm.
+ * demo of remark-gfm and rehype-raw working together and separate available [here](https://remarkjs.github.io/react-markdown/)
  * If you just want to render to plain [CommonMark](https://commonmark.org/), set the dialect to `'common-mark'`
  */
 export function MarkdownContent(props: Props) {
-  const { content, dialect = 'gfm', linkTarget } = props;
+  const { content, dialect = 'gfm', linkTarget, html = false } = props;
```

Comment:
Thoughts on calling it `trustHtml` or something like that? Just to make it clear that there's an additional level of trust that's needed. I'm assuming that this does not open up for script execution in any way right? But rather things like clickjacking through some creative styling.

To be clear as well this option is separate from the dialect right? so supports both common-mark and gfm

Reply:
Great points I also had to ponder and happy to talk more about. What did drive this implementation was the assumption of a trusted environment:
- totally agree to add something like `trustedHtml` to the prop name.  ReactMarkdown calls out that any HTML [would need to be in a trusted atmosphere to be totally safe](https://github.com/remarkjs/react-markdown#appendix-a-html-in-markdown).  Script execution shouldn't be able to happen, but clickjacking yes.  On that note, we can consider using rehype-sanitize. However, going that route does take away some features like being able to click ids and autofocus to parts of a page and doesn't exactly do what github.com actually supports, so the point of using it was defeated for the feature set this extension would try to accomplish.  
- This does support both dialects currently, which made this extension actually easier.

I'm happy to go with whatever you'd prefer.